rusoto/services/waf-regional/src/generated.rs

```diff
@@ -2091,6 +2091,96 @@ pub struct UpdateXssMatchSetResponse {
     pub change_token: Option<String>,
 }
 
+/// <p>The name specified is invalid.</p>
+#[derive(Default, Debug, Clone, PartialEq)]
+pub struct WAFDisallowedNameException {
```

Comment:
Not sure if it's too late to change this, but if we wanted to follow the Rust style guide recommendations the correct casing for this would be `WafDisallowedNameException`.

From [the section on naming](https://doc.rust-lang.org/1.0.0/style/style/naming/README.html):

> In CamelCase, acronyms count as one word: use Uuid rather than UUID. In snake_case, acronyms are lower-cased: is_xid_start.